test/unit/legacy-tests.js

```diff
@@ -25,7 +25,7 @@ module.exports = [
   'EnrollYubikey_spec.js',
   'ForgotPassword_spec.js',
   'IDPDiscovery_spec',
```

Comment:
Do you need to update path for other specs? or refactor the karma test config a bit so you dont have to change every entry here?

Reply:
These are run through a simple regex so it does not matter what their path, as long as it's not ambiguous. LoginRouter needs a more specific path because it contains the name of another test (BaseLoginRouter)